README.md

```diff
@@ -127,3 +127,11 @@ For development purposes requirement to authenticate can be turned off by settin
 environment variable TMP_PERMISSIONS_DISABLED to True.
 
 ![Data model visualization](tilavarauspalvelu_visualized.svg)
+
+## Deployed application settings & static files
+
+Contarary to more common set-ups, this application does not have a reverse proxy serving static files. Instead, static files are served by both Django and wsgi.
```

Comment:
Typo in `Contrary`

Reply:
Oh no! I should start reading what I write more often :sweat_smile: